CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Do not drop zero trace/span id spans in the jaeger conversion (#7946)
 - Upgrade to use semantic conventions 1.6.1 (#7926)
 - `dynatraceexporter`: Validate QueueSettings and perform config validation in Validate() instead (#8020)
+- `hostmetricsreceiver`: Add cpu.utilization metrics to cpu scrapper (#7130)
```

Comment:
thanks @dmitryax and sorry,  I don't know how many times I have changed this CHANGELOG and it was still not ok :(

Reply:
Hi @dmitryax , I fixed conflicts with CHANGELOG as there were more conflicts :) Is there anything else from my side that I should do?